docs/src/external_mctp_vdm_cmds.md

```diff
@@ -0,0 +1,508 @@
+# External MCTP VDM Commands Spec
```

Comment:
I think this needs to be linked in `SUMMARY.md` for it to appear in the sidebar?